eplaunch/interface/frame_main.py

```diff
@@ -0,0 +1,965 @@
+from datetime import datetime
```

Comment:
New main frame code in Tk.  It's down from 1700 lines to less than 1000, but admittedly, some of those lines are now just organized in other files.  And also it still needs favorite folders and group runs.